client/src/pages/Profile/Profile.js

```diff
@@ -61,13 +89,51 @@ class Profile extends Component {
                                 <Link className="loginLink" to="/login"><Button className="loginBtn">Login</Button></Link>
                             </>
                         ) : (
-                            <img id="loadingIcon" src="./assets/images/loading.gif" alt="loading"/>
+                            <img id="loadingIcon" src="./assets/images/loading-wheel.gif" alt="loading"/>
```

Comment:
Did you want a different loading gif?

Reply:
I'm not sure what they were pointing at. it was giving an error and that loading-wheel image was the only one in the folder. we will let them figure it out.